projectDocs/dev/buildingNVDAOnAppveyor.md

```diff
@@ -0,0 +1,520 @@
+
+It is assumed that you have already read the [NVDA] page on [Contributing], in particular the section on the [GitHub process](https://github.com/nvaccess/nvda/wiki/Contributing#github-process), as you will need to have an [NVDA] fork already in place on your development computer.
+
+Furthermore, you should already be generally familiar with [git], [GitHub], and how to work with repositories.
+This article won't go into any of these basic areas in any detail, and will only summarize topics the reasonably experienced developer is likely to understand already.
+
+## Situations not covered in this article
+
+* This document assumes your NVDA coding workflow involves branching off of `master` (or another official branch such as `beta`); working on the code; and pushing the result to your NVDA fork.\
+We don't cover, and the modifications we make to appveyor.yml are not tested with, any workflow which involves issuing pull requests against your NVDA fork.\
+It may work, but results are unknown.
+* Currently, we disable elements related to code signing, and don't attempt it.\
+It is discussed in the [Self-signed Build](./selfSignedBuild.md) document, but not in the context of building forks on [Appveyor].
+
+## Overview
+
+We will undertake the following general steps:
+* Setup GitHub and local environments.
+* Prepare local repository for Appveyor.
+* Modify the Appveyor configuration already included with NVDA to work for unofficial purposes.
+* Tell Appveyor where to find the new repository.
+* Configure Appveyor to find the config file.
+* Try the first build!
+
+## Getting started
+
+If you haven't already, per the "Prerequisites" section above, fork a copy of the main [NVDA] repository into your GitHub account.
+You can call your fork "NVDA", or "my_nvda", or whatever you want.
+For the rest of this article, we will assume you have called it "nvda".
+
+Next, clone the fork to your local dev environment, and perform the other setup tasks recommended in the [Contributing] document.
+
+If you are using the "[gh]" tools for working with [GitHub] from the commandline, you can do both steps in one command with:
+
+```sh
+gh repo fork --clone=true --remote=true nvaccess/nvda -- --recurse-submodules
+```
+
+## Where to put the build file
+
+In order to actually build NVDA, Appveyor has to be given a set of instructions to follow.
+This is done using the `appveyor.yml` file, which is a text file in the [YAML] format.
+(More on the format and content later.)
+
+The most common placement for this file, is in the root directory of your repository.
+Unfortunately, since NV Access already uses Appveyor for official builds, it would be difficult to keep your build file in the root directory of your NVDA fork.This is because you will need to permanently change a few items in that file, and you wouldn't want that private build file "contaminating" your pull requests or diffs.
```

Comment:
These sentences should preferably be each on its own line. If not there is a missing space:
```suggestion
Unfortunately, since NV Access already uses Appveyor for official builds, it would be difficult to keep your build file in the root directory of your NVDA fork. This is because you will need to permanently change a few items in that file, and you wouldn't want that private build file "contaminating" your pull requests or diffs.
```

Reply:
Thanks @lukaszgo1, missed that one. They are all supposed to be on their own lines.